modules/ROOT/pages/access-source-message-attributes.adoc

```diff
@@ -14,7 +14,7 @@ Accessing transmission IDs in the translation maps enables end-to-end transmissi
 +
 * Mapping attributes to pass them to the back end
 +
-For example, if the back end must see the source filename or a specific HTTP header, you can access the filename or HTTP header and map the values to the translated message payload fields.
+For example, if the backend must see the source filename or a specific HTTP header, you can access the filename or HTTP header and map the values to the translated message payload fields.
```

Comment:
Is "see" correct? Seems anthropomorphic.

Reply:
I can't think of another word to use here.